File: resources/php/delete.php
<?php

require_once __DIR__ . '/bootstrap.php';

include "config.php";

try {
    // Create connection
    $conn = new mysqli($servername, $username, $password, $dbname);

    // Check connection
    if ($conn->connect_error) {
        throw new Exception("Connection failed: " . $conn->connect_error);
    }

    // Check if the form is submitted
    if ($_SERVER["REQUEST_METHOD"] == "POST") {
        if(isset($_POST['id']))  {

            $id = $_POST['id'];

            // Delete from users table (cascades to all related tables)
            $sqlUser = "DELETE FROM users WHERE userid = ?";
            $stmtUser = $conn->prepare($sqlUser);
            $stmtUser->bind_param("i", $id);
            $stmtUser->execute();

            if($stmtUser->affected_rows > 0) {
                echo 1; // success

            } else {
                echo "Something Went Wrong"; // failure
            }
            $stmtUser->close();
        }
    }

} catch (Exception $e) {
    echo  $e->getMessage(), "\n";
}

?>

File: resources/php/delete.py
from flask import request

from db import get_connection
from db_init import initialize_schema


def register_routes(app):
    @app.route("/resources/php/delete.py", methods=["POST"])
    def delete_php():
        conn = None
        try:
            conn = get_connection()
            initialize_schema(conn)
            if request.method == "POST" and "id" in request.form:
                uid = request.form.get("id")
                with conn.cursor() as cur:
                    cur.execute("DELETE FROM users WHERE userid = %s", (int(uid),))
                    affected = cur.rowcount
                conn.commit()
                if affected > 0:
                    return "1"
                return "Something Went Wrong"
            return ""
        except Exception as exc:
            return str(exc) + "\n"
        finally:
            if conn is not None:
                conn.close()

